packages/atomic/src/components/common/result-children/collection-guard.tsx

```diff
@@ -40,17 +38,13 @@ export const CollectionGuard: FunctionalComponent<CollectionGuardProps> = (
     );
   }
 
-  if (!moreResultsAvailable && !hasChildren) {
+  if (!hasChildren) {
     return noResultText.trim().length ? (
       <p part="no-result-root" class="no-result-root my-3">
         {noResultText}
       </p>
     ) : null;
   }
```

Comment:
This seem weird but it seems like in the case of folded results, there is no way of knowing if there can be more results available so we just put the value as "true" at the collection creation. https://github.com/coveo/ui-kit/blob/7c110bc8357043fc1ddd368ddb333997fef41717/packages/headless/src/features/folding/folding-slice.ts#L120